moabb/datasets/upper_limb.py

```diff
@@ -102,6 +102,9 @@ def _get_single_subject_data(self, subject):
                 raw = read_raw_gdf(
                     path, eog=eog, misc=range(64, 96), preload=True, verbose="ERROR"
                 )
+                # convert the unit to microvolt
+                raw._data *= 1e-6
+
```

Comment:
The code is converting from microvolt to volt. I think the code is correct and we always want to return raw data in volt.  But the comment says the opposite

Reply:
Wrong comment, sorry